src/serverMain/kotlin/com/studo/campusqr/endpoints/LocationManagement.kt

```diff
@@ -139,6 +151,7 @@ suspend fun ApplicationCall.visitLocation() {
   }
 
   runOnDb {
+    // Maybe ensure that user with email can check-in only once?
```

Comment:
is this a TODO? if yes, we should resolve it. if no i don't understand this comment